lib/omb-prompt-base.sh

```diff
@@ -538,6 +539,16 @@ function _omb_prompt_get_python_env {
   [[ $python_env ]]
 }
 
+## @fn _omb_prompt_get_spack_env
+##   @var[out] spack_env
+##   @exit
+function _omb_prompt_get_spack_env {
+  spack_env=
+  [[ ${SPACK_ENV-} ]] || return 1
+  [[ ${OMB_PROMPT_SHOW_PYTHON_VENV-} == true ]] || return 1
```

Comment:
```suggestion
  [[ ${OMB_PROMPT_SHOW_SPACK_ENV-} == true ]] || return 1
```

Reply:
Also, I think we should test this before testing `${SPACE_ENV-}`

Reply:
What option would be the best for oh-my-bash ?
Intuitively, I'd say that not changing the theme would be better (avoiding changing all themes), but the users will have to activate by hand

Reply:
Now you've added a mechanism to opt-in `OMB_PROMPT_SHOW_SPACK_ENV`, so we can include the spack environment information in the prompt of the `font` theme without affecting the existing users. I re-added the changes to the `font` theme in commit 0559dc7.

Reply:
ok great! Thanks :)